### PR TITLE
Update Contribution Guidelines

### PR DESCRIPTION
**Pull request by nbayramberdiyev**
This PR;
- corrects coding standard, CI workflows and commands in `CONTRIBUTING.md` file,
- corrects min. PHP version in `README.md` file,
- ~~ignores IDE-related files from tracking by Git.~~

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ vendor/
 *.cache
 infection-log.*
 build/
+.idea/
```
Comment (juliangut) [inline .gitignore]: Please remove IDE specific paths
These should be ignored globally in `~/.gitignore`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,17 +6,16 @@ Make your contributions through Pull Requests
...
+- Follow **[PSR-12](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-12-extended-coding-style-guide.md)** coding standard
 - **Unit test everything** and run the test suite
 - Try not to bring **code coverage** down
 - Keep documentation **updated**
 - Just **one pull request per feature** at a time
-- Check that **[Travis CI](https://travis-ci.org/juliangut/slim-php-di)** build passed
+- Check that **[GitHub Actions](https://github.com/juliangut/slim-php-di/actions)** build passed
 
-Composer scripts are provided to help you keep code quality and run the test suite:
+Makefile commands are provided to help you keep code quality and run the test suite:
```
Comment (juliangut) [inline CONTRIBUTING.md]: change "commands" into "targets"

**Comment by juliangut** [comment]
Hi @nbayramberdiyev thanks for the corrections, please review my comments and I'll merge the PR

**Comment by nbayramberdiyev** [comment]
Done! One more thing, you mentioned `\Interop\Container\ContainerInterface` in `README.md` which was [officially deprecated](https://github.com/container-interop/container-interop). I think we should replace it with `Psr\Container\ContainerInterface`. [See here](https://php-di.org/doc/migration/6.0.html#container-interop-and-psr-11).

**Comment by juliangut** [comment]
You are right, it is a typo in readme, it definitely should state `\Psr\Container\ContainerInterface` I'll fix it
Thanks

**Pull request merged**